scripts/sonic-sync-mode

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/python
+import argparse
+from swsscommon import swsscommon
+
+def main():
+    parser=argparse.ArgumentParser(description=
+          'Configure enable or disable synchronous mode between orchagent and syncd',
+          formatter_class=argparse.RawTextHelpFormatter,
+          epilog=
+'''
+sudo needed for configuring synchronous mode
+swss restart required to apply the configuration
+Options to restart swss and apply the configuration:
+    1. config save
```

Comment:
>config save [](start = 7, length = 11)

`config save` also has `-y`. Do you want using?

Reply:
Added `-y` in the comment